oap-server/server-starter/src/main/resources/meter-analyzer-config/network-profiling.yaml

```diff
@@ -27,35 +27,51 @@ expPrefix: |-
     }
     tags[prefix + '_process_id'] = ProcessRegistry.generateVirtualRemoteProcess(tags.service, tags.instance, tags[prefix + '_address'])
   })
+  .forEach(['component'], { key, tags ->
+    String result = ""
+    // protocol are defined in the component-libraries.yml
+    String protocol = tags['protocol']
+    String ssl = tags['is_ssl']
+    if (protocol == 'http' && ssl == 'true') {
+      result = '129'  // https
+    } else if (protocol == 'http') {
+      result = '49'   // http
+    } else if (ssl == 'true') {
+      result = '130'  // TLS
+    } else {
+      result = '50'   // rpc
```

Comment:
Should this be `tcp=110`? RPC seems not accurate?

Reply:
Yes, I think tcp is more accurate. Updated.